### PR TITLE
Added font coloring to GoalForm > GoalInput

### LINKED ISSUE #813: [BUG] Font Color error in Goals section on dashboard
### Describe the bug

Whatever we add in Goals section the font color is white on white background making it hard to visible 
goals

### To Reproduce

Go to Dashboard 
Go to Goals 
Enter anything

### Expected Behavior

_No response_

### Screenshot/ Video

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by alvarotorrestx**
## Fixes Issue

Fixes #813 

## Changes proposed

Added font color to input text which matches website theme

## Screenshots

image
image

## Note to reviewers

I found some additional bugs that are on this page. Am I allowed to create issues for these? If not, can issues be created and may I please be assigned to them?

They are as follows:

image
image
The add button is overflowing outside the input box

image
image
There is heavy overflow on the goals that are added also

## Code of Conduct

-   [x] By submitting this pull request, I confirm I've read and complied with the [CoC](https://github.com/thecyberworld/TheCyberHUB/blob/dev/CODE_OF_CONDUCT.md) 🖖

## Check List (Check all the applicable boxes) 

-   [x] My code follows the code style of this project.
-   [ ] My change requires changes to the documentation.
-   [ ] I have updated the documentation accordingly.
-   [x] All new and existing tests passed.
-   [x] This PR does not contain plagiarized content.
-   [x] The title of my pull request is a short description of the requested changes.

---

You can also join our [Discord](https://discord.gg/QHBPq6xP5p) community.
Feel free to check out other cool repositories of the [Thecyberworld](https://github.com/thecyberworld).
Join the Thecyberworld GitHub Organisation by raising an [issue](https://github.com/thecyberworld/Support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) (you will be sent an invitation).

### CODE REVIEW

**Comment by ArkadiK94** [comment]
@alvarotorrestx I know that this was not specified in the issue, but could you change the color of the button as well to be visible? At the end of the input box there is a '+' button

**Comment by alvarotorrestx** [comment]
> @alvarotorrestx I know that this was not specified in the issue, but could you change the color of the button as well to be visible? At the end of the input box there is a '+' button

For sure

**Comment by alvarotorrestx** [comment]
> @alvarotorrestx I know that this was not specified in the issue, but could you change the color of the button as well to be visible? At the end of the input box there is a '+' button

Change has been committed to the same branch 👍

**Comment by kabir0x23** [comment]
hey, can u add screenshots, so we can review it better, and fast

File: src/components/Dashboard/OldDashbaord/GoalSetter/GoalForm.jsx [hunk 1]
```diff
@@ -46,7 +47,10 @@ const GoalForm = () => {
                         onChange={(e) => setText(e.target.value)}
                         placeholder="What's your goal?"
                     />
-                    <button className={"btn btn-block"} type={"submit"}>
+                    <button
+                        className={"btn btn-block !bg-[#ff6b08] !text-[#1c1c1c] hover:opacity-[.95]"}
+                        type={"submit"}
```
[the remark this answers is not in the document]
Comment (kabir0x23) [reply src/components/Dashboard/OldDashbaord/GoalSetter/GoalForm.jsx:52]: nice, right

but issue is this goal app is just mixuture of styled .css tailwinds

anyway, we are replacing this one with task app in future

[the remark this answers is not in the document]
Comment (alvarotorrestx) [reply src/components/Dashboard/OldDashbaord/GoalSetter/GoalForm.jsx:52]: Sure, I apologize. I’ll update using the styled component today and will submit with screenshot.

Do you want the input bar to be restyled to match your color scheme such as the input on notes portion of dashboard?

The button is also orange on those portion of the app.

[the remark this answers is not in the document]
Comment (kabir0x23) [reply src/components/Dashboard/OldDashbaord/GoalSetter/GoalForm.jsx:52]: > Sure, I apologize. I’ll update using the styled component today and will submit with screenshot.
> 
> Do you want the input bar to be restyled to match your color scheme such as the input on notes portion of dashboard?
> 
> The button is also orange on those portion of the app.

u can improve the design in any way,. 

> Do you want the input bar to be restyled to match your color scheme such as the input on notes portion of dashboard?

orange wil work, if it is looking good on styles

**Comment by alvarotorrestx** [comment]
I just resubmitted a redesign using the styled component. Can you please review and let me know what you think?

Here are some screenshots:
image
image

The design is reflecting a similar design to what you all have already come up with.
image

Thanks

**Review by kabir0x23 — APPROVED** [review]
looking great, thanks for the contributions

**Pull request merged**